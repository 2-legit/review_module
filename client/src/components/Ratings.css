.sub-ratings {
  display: flex;
  flex-wrap: wrap;
  width: 594px;
  justify-content: space-between;
  padding-top: 15px;
  padding-bottom: 15px;
  border-bottom: solid #efefef 1px;
}

.rating {
  flex: 0 0 280px;
  display: flex;
  justify-content: space-between;
}

