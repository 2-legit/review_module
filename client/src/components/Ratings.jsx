import React from 'react';

//In the render here, we will be passed in the ratings from
// the App state. From there, we want to get the avg rating.

class Ratings extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div>
        Ratings will be here!
      </div>
    )
  }
};

export default Ratings;
