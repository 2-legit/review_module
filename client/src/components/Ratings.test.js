import React from 'react';
import { shallow, mount, render } from 'enzyme';

import Ratings from './Ratings'

describe('Ratings Test Suite', () => {
  test('Should have a matching snapshot', () => {
    const component = shallow(<Ratings />);
    expect(component).toMatchSnapshot();
  })

})
