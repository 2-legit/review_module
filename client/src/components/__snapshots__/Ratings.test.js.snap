// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ratings Test Suite Should have a matching snapshot 1`] = `ShallowWrapper {}`;
